arcgis-rest-api/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for arcgis-to-geojson-utils 10.4
```

Comment:
We get warnings about identical project names, so I'd like to handle this before merging.